src/cs-utils/FrameTimings.cpp

```diff
@@ -178,7 +180,7 @@ void TimerQueryPool::end(std::string const& name) {
 
   auto iter = mQueryRanges.find(name);
   if (iter == mQueryRanges.end()) {
-    std::cerr << "WARN: TimerQueryPool::End, unknown key: " << name << '\n';
+    spdlog::error("Failed to end timer query: Unknown key '{}'!", name);
```

Comment:
The original was a warning. Should this too?
```suggestion
    spdlog::warn("Failed to end timer query: Unknown key '{}'!", name);
```